src/Rule/Rule.php

```diff
@@ -94,7 +89,7 @@ public static function parse(ParserState $parserState): Rule
         if ($parserState->getSettings()->bLenientParsing) {
             while ($parserState->comes('\\')) {
                 $parserState->consume('\\');
-                $rule->addIeHack($parserState->consume());
+                $parserState->consume();
                 $parserState->consumeWhiteSpace();
             }
         }
```

Comment:
I think this whole code block is parsing the IE hack(s).  And the idea is that IE will treat them as valid rules whilst other browsers will discard them.  There's a [mention in the readme that IE hacks are only supported in lenient mode](https://github.com/MyIntervals/PHP-CSS-Parser#contributorsthanks-to).  So, on that basis, I think the entire `if` statement should be removed, so that invalid declarations with IE hacks are discarded rather than kept.